ehsm_kms_service/router.js

```diff
@@ -180,6 +192,12 @@ const router = async (p) => {
     case key_management_apis.DeleteAllKey:
       deleteALLKey(appid, res, DB)
       break
+    case key_management_apis.EnableKey:
+      enableKey(appid, payload, res, DB)

```

Comment:
Could you also check other funtions e.g. @line 189:
    case key_management_apis.DeleteKey:
      deleteKey(payload, res, DB)
      break

Is appid also required?

Reply:
Please submit a new patch to fix the issue in DeleteKey.

Reply:
Ok,I will fix it.